From 3fd3f115fac45e68630ee4bb128693a7083528c7 Mon Sep 17 00:00:00 2001
From: Pierre-Hugues Husson <phh@phh.me>
Date: Tue, 11 Aug 2026 09:13:12 +0000
Subject: [PATCH] Include vendor.samsung.hardware.radio-V2.2-java HIDL in base
 libs

[Port to A16] Inserted after android.hardware.radio-V1.6-java (context refreshed for V5 radio libs).

Change-Id: Ie25f6b545dccaa3444aaf32a981bf1a345856036
---
 Android.bp | 1 +
 1 file changed, 1 insertion(+)

diff --git a/Android.bp b/Android.bp
index 1ede4ccf8731..bb08a31fac55 100644
--- a/Android.bp
+++ b/Android.bp
@@ -227,6 +227,7 @@ java_library {
         "android.hardware.gnss-V2.1-java",
         "android.hardware.health-V1.0-java-constants",
         "android.hardware.radio-V1.6-java",
+        "vendor.samsung.hardware.radio-V2.2-java",
         "android.hardware.radio.data-V5-java",
         "android.hardware.radio.ims-V4-java",
         "android.hardware.radio.messaging-V5-java",
-- 
2.43.0

